doc/classes/PackedByteArray.xml

```diff
@@ -379,6 +379,12 @@
 				Returns [code]true[/code] if the array is empty.
 			</description>
 		</method>
+		<method name="pop_back">
+			<return type="int" />
+			<description>
+				Removes and returns the last element of the array.
```

Comment:
These should describe what happens if the array is empty

Reply:
Hmm, i guess it would be good to make it not error then.

Reply:
Although then its slightly confusing to get 0 when your array is empty

Reply:
Yeah no ill say it errors and returns $default value